# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/PHttpTransaction.ipdl
# Commit: 96bd98319c62
# Full Hash: 96bd98319c6242a9b05dc4723b928e88537424e1
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-08-26 03:24:37
# Regressor Bug: 1652667
# File Overlap Count: 1
# Description:
#   Bug 1652667 - P1: Add telemetry for when an HTTPSSVC record is received r=dragana
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D87844
# ==============================================================================

diff -r b61a789151df -r 96bd98319c62 netwerk/protocol/http/PHttpTransaction.ipdl
--- a/netwerk/protocol/http/PHttpTransaction.ipdl	Tue Aug 25 17:29:21 2020 +0000
+++ b/netwerk/protocol/http/PHttpTransaction.ipdl	Tue Aug 25 17:29:32 2020 +0000
@@ -46,7 +46,8 @@
                        uint8_t[]                  dataForSniffer,
                        nsCString?                 altSvcUsed,
                        bool                       dataToChildProcess,
-                       bool                       restarted);
+                       bool                       restarted,
+                       uint32_t?                  HTTPSSVCReceivedStage);
   async OnTransportStatus(nsresult status,
                           int64_t progress,
                           int64_t progressMax,